osu.Game.Rulesets.Taiko/Skinning/Legacy/LegacyInputDrum.cs

```diff
@@ -91,7 +90,7 @@ protected override void Update()
 
             // Relying on RelativeSizeAxes.Both + FillMode.Fit doesn't work due to the precise pixel layout requirements.
             // This is a bit ugly but makes the non-legacy implementations a lot cleaner to implement.
-            content.Scale = new Vector2(DrawHeight / content.Size.Y);
+            Scale = new Vector2(Parent.DrawHeight / Size.Y);
```

Comment:
This change is taking it from "a bit ugly" to "really ugly". The parent isn't required to really even be there, in general.

Does this need to happen? Can't the input drum just autosize or something, rather than doing all of this moving of the hardcoded size up a level?